app/facades/google_user_facade.rb

```diff
@@ -0,0 +1,7 @@
+class GoogleUserFacade
+  def self.user(user_info)
+    user_attributes = GoogleUsersService.create_or_find_user(user_info)
+    formatted = user_attributes[:data][:attributes]
+    GoogleUser.new(formatted)
```

Comment:
Same here, the resource creation is happening in the backend so I think we'll only be testing that the correct record is being returned.